composer.json

```diff
@@ -53,5 +53,11 @@
     },
     "config": {
         "preferred-install": "dist"
-    }
+    },
+    "repositories": [
+        {
+            "type": "git",
+            "url": "ssh://git@github.com/SocialiteProviders/Microsoft-Graph.git"
```

Comment:
@pavril `composer install` and `composer update` seem to work fine on my computer. Weird.